templates/events/add_event.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'layout.html' %}
+{% block content %}
+<div class="container">
+    <div class="py-3 mt-3 text-center">
+        <h2 class="font-weight-bolder">Add new event</h2>
+    </div>
+    <div class="row justify-content-center">
+        <div class="col-md-8 shadow-lg p-5 mb-5 bg-white rounded">
+            <form method="POST" enctype="multipart/form-data" class="needs-validation" novalidate>
+                {% csrf_token %}
+                <div class="row">
+                    <div class="col-md-6 mb-3">
+                        <label for="title">Event name</label>
+                        <input name="title" type="text" class="form-control" id="title" required />
+                        <div class="invalid-feedback">Event name is required.</div>
+                    </div>
+                    <div class="col-md-6 mb-3">
+                        <label for="schedule">Schedule</label>
+                        <input name="schedule" class="form-control" type="datetime-local" value="2011-08-19T13:45:00" id="schedule">
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-6 mb-3">
+                        <label for="venue">Venue</label>
+                        <input name="venue" class="form-control" type="text" id="venue" required />
+                        <div class="invalid-feedback">Please enter a venue of your event.</div>
+                    </div>
+                    <div class="col-lg-6 mb-3">
+                        <label for="contact">Contact No. </label>
+                        <input name="contact" type="tel" pattern="[0-9]{10}" class="form-control" id="contact" required />
+                        <div class="invalid-feedback">Valid Contact No. (10 Digits) is required.</div>
+                    </div>
+                </div>
+                <div class="mb-3">
+                    <label for="description">Event Description</label>
+                    <textarea name="description" class="form-control" id="description" rows="3"></textarea>
+                </div>
```

Comment:
I think 5 rows would be better instead of 3. Also, event description is required.